utils/storybook/MarkdownWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import 'github-markdown-css';
+import EmojiConvertor from 'emoji-js';
+import Markdown from 'markdown-to-jsx';
+
+const emoji = new EmojiConvertor();
+
+interface MarkdownWrapperProps {
+  content: string;
+}
+
+export default class MarkdownWrapper extends React.Component<MarkdownWrapperProps> {
```

Comment:
Can we just call this `Markdown`? The end "user" doesn't really need to know it's a wrapper.

Reply:
Updated 👍